ant-cli/src/commands.rs

```diff
@@ -66,8 +71,6 @@ pub enum FileCmd {
         addr: String,
         /// The destination file path.
         dest_file: String,
-        /// Optionally specify the quorum for the verification of the download.
-        read_quorum: Option<ResponseQuorum>,
```

Comment:
It makes no sense to add a quorum option for download of chunks as 1 is enough and there is no verification, so removed as the example is irrelevant.

Reply:
This is used with our performance monitoring system.

Reply:
Restored!